tests/add-user-ui.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { test, expect } from "@playwright/test";
+
+test("has title", async ({ page }) => {
+  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
+
+  // Expect a title "to contain" a substring.
+  await expect(page).toHaveTitle("TS Trainee course");
+});
+
+test("Create button UI", async ({ page }) => {
+  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
+  const el = page.getByRole("button", { name: "Create" });
+  await expect(el).toBeVisible;
+  await expect(el).toHaveClass("btn btn-primary");
+  await expect(el).toHaveCSS("background-color", "rgb(13, 110, 253)");
```

Comment:
What is the magic numbers? 
It will be better to create an enum for example with appropriate names and values and afterward use it in your code. 

Example: 
export enum Colors {
  VividBlue = 'rgb(13, 110, 253)',
}